### ISSUE TITLE
SyntaxError when test using jest

### ISSUE THREAD

**Issue by HyunmoAhn** [inorganik/countUp.js#218]
## Concept
- [x] Bug

## Version
CountUp.js version: v2.x

## Description
I can use `countUp.js` actual environment like browser.
But I have an error when unit test using `jest`.
![image](https://user-images.githubusercontent.com/23732795/55203885-6e00a080-5210-11e9-8caf-0c7473b8cfbb.png)

It show me that this error is **syntax error**!
It maybe occur because It is wrong of my jest configure.
But `countUp.js` v1.x is work correctly, doesn't occur `syntax error`.

I leave my `jest.config.js` 
```
module.exports = {
  moduleFileExtensions: [
    'js',
    'jsx',
    'json',
    'vue'
  ],
  transform: {
    '^.+\\.vue$': 'vue-jest',
    '.+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$': 'jest-transform-stub',
    '^.+\\.jsx?$': 'babel-jest',
  },
  "transformIgnorePatterns": [
   "/node_modules/(?!lodash-es).+\\.js$",
  ],
  moduleNameMapper: {
    '^@/(.*)$': '<rootDir>/src/$1'
  },
  snapshotSerializers: [
    'jest-serializer-vue'
  ],
  testMatch: [
    '**/tests/unit/**/*.spec.(js|jsx|ts|tsx)|**/__tests__/*.(js|jsx|ts|tsx)'
  ],
  coveragePathIgnorePatterns: [
    "/node_modules/",
    "/tests/"
  ],
  testURL: 'http://localhost/'
}

```
Please tell me. why does occur syntax error only `countUp.js` v2.x. ?

**Comment by HyunmoAhn** [comment]
Answer by myself...

It occur `jest.config.js`

```
  "transformIgnorePatterns": [
    "/node_modules/(?!countup\\.js).+\\.js$",
  ],
```

Basic configure of jest recommend that doesn't transform modules on `/node_modules`.
So,  `countUp.js` was not target of transfiling on `babel-jest`. Above configure make exception of transfiling.

**I have question.**

Why do I transfile `countUp.js`? It don't need to `countUp.js` version 1.x.

**Comment by garrettmaring** [comment]
I encountered this issue as well. The last line of the countUp.js (and .min) has the `export` keyword. That's invalid syntax as far as I know.

**Comment by inorganik** [comment]
@HyunmoAhn I found the [same error](https://github.com/inorganik/countUp.js-angular2/issues/28#issuecomment-484235088) myself in countUp-angular2 and solved the same way you did. For whatever reason, babel doesn't like ES6 modules. (I found [this](https://stackoverflow.com/q/33604470/591487)) So you just need to tell the jest transformer to ignore the file. The old CountUp (1.x) was exported as an es2015 UMD module, so it didn't use import/export.

@garrettmaring `export` is valid ES6 syntax. CountUp is transpiled from typescript and exported as an ES6 module.
